### DISCUSSION TITLE
Queries and mutations GraphQL

### DISCUSSION THREAD

**Discussion by AlexisHeap** [saleor/saleor-platform#235] [Q&A]
Hello team, where can I make GraphQL queries and mutations inside the saleor-platform project? I am running it locally OR which is the procedure-idea? thank you.

**Comment by peelar** [comment]
Hello @AlexisHeap 👋 

If you want to run Saleor and its services locally, `saleor-platform` is the best bet. It boots up Saleor API at `http://localhost:8000`. You can find the GraphQL playground at [http://localhost:8000/graphql/](http://localhost:8000/graphql/).

You can also leverage a free developer [Saleor Cloud](https://cloud.saleor.io/org/) plan. Each project will also have the same GraphQL playground, where you can try out your queries and mutations.
